### DISCUSSION TITLE
Cannot enable RLS for spatial_ref_sys table

### DISCUSSION THREAD

**Discussion by cjacky475** [supabase/supabase#26302] [Questions] [answered]
Hello, when I enabled PostGIS for public schema the role for the spatial_ref_sys was assigned to supabase_admin, however, through dashboard I cannot access this role. How to enable RLS for my table then? Cheers.

**Comment by GaryAustin1** [comment, marked as answer]
Not sure they should be in the public schema.  The default from the UI is extensions schema so they can't be accessed from the API.
https://github.com/orgs/supabase/discussions/19143

You can't modify tables created by supabase_admin.
